accepted/2025/cli-acquisition-tool.md

```diff
@@ -0,0 +1,224 @@
+# .NET SDK Acquisition and Management Tool
+
+**Owner** [Chet Husk](https://github.com/baronfel) | [Daniel Plaisted](https://github.com/dsplaisted)
+
+Users of the .NET SDK today face a wide array of choices when getting our tooling, with multiple distribution channels that release at different cadences.
+While these options serve specific needs, several important user cohorts are not well served and have asked for a more unified approach to getting started with our tooling.
+
+
+This proposal introduces a standalone install and SDK management tool (tentatively named `dnup`) that provides a unified interface for acquiring and managing .NET tooling across all supported platforms, focusing on user-local installations that can be managed (largely) without elevation.
+
+This tool is especially critical for users that are
+
+* new users, especially those from the Node or Rust ecosystems
+* Linux users who typically use packages from their distro, as not all distro maintainers support feature bands beyond 1xx
```

Comment:
One thing I have heard time and time again is that for a subset of enterprise customers, their processes allow for use of packages from the native package feeds.  Anything beyond them requires special approval if at all.  This is to say acquiring components outside the native feeds is significantly harder if not prohibited.  dnup does not change this, unless dnup is written in such a way to acquire from the package feeds.

Reply:
You're right - `dnup` doesn't change this situation. `dnup` is explicitly not targeting package feeds at all, users that need fleetwide or admin-level management of systems would still rely on those mechanisms.